### PR TITLE
Improve API Features for using in 'rlottie preview'

### PR DESCRIPTION
**Pull request by msnodeve**
> Summary of features in rlottie

1. **Add basic Property API(function)**
    1. FillColor
    2. FillOpacity
    3. StrokeColor
    4. StrokeOpacity
    5. StrokeWidth
    6. TrPosition
    7. TrRotation
    8. TrAnchor
    9. TrScale
2. **Add slow functions**
    1. Change properties smothly from start value to end value

### CODE REVIEW

File: src/wasm/rlottiewasm.cpp [hunk 1]
```diff
@@ -41,6 +41,157 @@ class __attribute__((visibility("default"))) RlottieWasm {
...
+    }
+
+    void setStrokeWidth(std::string key, float width) {
+        mPlayer->setValue<rlottie::Property::StrokeWidth>(key,
+                [width](const rlottie::FrameInfo& info) {
+                    return width;
+                });
+    }
+
+    void setTrAnchor(std::string key, float x, float y){
```
Comment (rlottie) [inline src/wasm/rlottiewasm.cpp:80]: no need to pass a lambda when we have api which takes a static value. Same for below functions as well.
mPlayer->setValue<rlottie::Property::StrokeWidth>(key, width) will do the job.

**Review by rlottie — CHANGES_REQUESTED** [review]

File: src/wasm/rlottiewasm.cpp (hunk 1 above)
Comment (msnodeve) [reply src/wasm/rlottiewasm.cpp:65]: Oh, Thank you. 
I'll modify the code.

**Comment by msnodeve** [comment]
We removed all the features except the required features to be used in the preview.
And we removed the lambda function.
Please review it.

**Review by hermet — APPROVED** [review]
Looks fine by me.
@smohantty  any opinions?

File: src/wasm/rlottiewasm.cpp [hunk 2]
```diff
@@ -105,5 +256,26 @@ EMSCRIPTEN_BINDINGS(rlottie_bindings)
         .constructor(&RlottieWasm::create)
         .function("load", &RlottieWasm::load, allow_raw_pointers())
         .function("frames", &RlottieWasm::frames)
-        .function("render", &RlottieWasm::render);
-}
+        .function("render", &RlottieWasm::render)
+        .function("set_fill_color", &RlottieWasm::setFillColor)
```
Comment (rlottie) [inline src/wasm/rlottiewasm.cpp:153]: use CamelCase function ex: setFillColor  instead of ste_fill_color . and the rest of the Apis.

**Review by rlottie — COMMENTED** [review]
Also Squash both the commit to make 1 commit .

File: src/wasm/rlottiewasm.cpp (hunk 2 above)
Comment (msnodeve) [reply src/wasm/rlottiewasm.cpp]: Thank you!, I will make you new pull request! :D

**Pull request closed without merging**